### ISSUE TITLE
Questions about FeaturePlot in seurat v3.0

### ISSUE THREAD

**Issue by sunshine-lp0** [satijalab/seurat#1014]
Hi,
I found that "FeaturePlot" function in seurat v3.0 consumed more time than  seurat v2.3( I used 40 genes). Also, it's difficult to open the pdf file. Is it because I use too many genes?

**Comment by andrewwbutler** [comment]
Hi,

We haven't noticed a significant difference in plotting time between the two versions. The time to plot will scale with both the number of cells (dots in the plot) and the number of genes (number of plots). What is the scale of data you're working with? 

Likely the issue with opening the PDF file is due to a large number of cells and is going to be limited by the resources of the computer/tool you're trying to view the PDF with. We introduced the `vector.friendly` argument in v2 (see the [MCA vignette](https://satijalab.org/seurat/mca) for an example), which rasterizes the points in the plot while leaving the axes/text as vectors to make viewing plots with large numbers of cells faster and more manageable.  This has been replaced by `AugmentPlot` in v3 so the equivalent function calls could look something like: 
```R
p <- FeaturePlot(object = object,  features = c("F1", "F2"), combine = FALSE)
p <- lapply(X = p, FUN = AugmentPlot)
CombinePlots(plots = p)
```
